xblock/fields.py

```diff
@@ -146,6 +148,24 @@ def scopes(cls):
         return [cls.NONE, cls.ONE, cls.ALL]
 
 
+class SharedUserScope(object):
+    """
+    Enumeration of shared scopes.
+
+    To be continued.
+    """
+    ONE = Sentinel('SharedUserScope.ONE')
+    GROUP = Sentinel('SharedUserScope.GROUP')
```

Comment:
We don't have groups yet. I'd hold off on implementing this until we do, perhaps? I'm actually not entirely clear on why we need multiple scopes here in the first place -- shared implies it is not a user_state_summary, but per-user for now.